examples/CONFIG/bin/README.md

```diff
@@ -2,6 +2,5 @@
 
 Place the `walrus` client binary for your system in this directory.
 
-See [the installation
-documentation](https://mystenlabs.github.io/walrus-docs/usage/setup#installation) for how to obtain
-the binary.
+See [the installation documentation](https://docs.walrus.site/usage/setup#installation) for how to
```

Comment:
I am in two minds about this: it means that of Walrus is down we cannot navigate docs? Is this a good idea?

Reply:
This is a very good point and something we need to reach a conclusion about as we already point to docs.walrus.site in many places.

I see two different approaches here:
1. Actually use Walrus and WS for our resources but then make sure we are alerted quickly if there are any issues. Brief downtimes (e.g., during re-deployments) are probably not an issue at this point.
2. Treat Walrus as an unreliable system and point to our GH Pages site for docs and other important information and/or mention it as a fallback.

I'm leaning towards (1) as this also demonstrates our own confidence in the system.

In any case, we can put this on hold for now.

Reply:
@gdanezis Given that Walrus Sites is more reliable than GitHub Pages, how about we merge this in? 😁